src/typings/uiv.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module 'uiv' {
```

Comment:
Are we switching to using typescript for this repo's vue files?

Reply:
I don't think we should update any components defined in this repo to use TS (right now), but if we're going to import components from Synapse then I think it'd be best for both `dev-center` and `web` to be as flexible as possible and accept TS components.

Reply:
Sounds good!